src/patternfly/components/Masthead/masthead-brand.hbs

```diff
@@ -0,0 +1,15 @@
+<{{#if masthead-brand--type}}{{masthead-brand--type}}{{else}}a{{/if}} class="pf-c-masthead__brand{{#if masthead-brand--modifier}} {{masthead-brand--modifier}}{{/if}}"
+  {{#if masthead-brand--href}}
+    href="{{masthead-brand--href}}"
+  {{else}}
```

Comment:
This creates a problem if `masthead-brand--type` is passed, you can't remove `href`. An obvious solution isn't sticking out so I'd be in favor of just making this work for our default use case and update it later if we need to.

Also should the default be `href="/"`, so that it's a location? `#` will add that to the address bar, too.